### DISCUSSION TITLE
Longhorn Volume Actualsize is bigger then pod's usage

### DISCUSSION THREAD

**Discussion by Batyray** [longhorn/longhorn#9945] [Q&A]
Hello right now I am using longhorn for my pv in my kubernetes cluster. One of the technology we use called restate uses 3.4 gb inside the pod but the actualize used in the longhorn console is much more bigger than the pod's usage. You can see the details in screenshots. Have you ever faces with such issue before ? 
<img width="800" alt="image" src="https://github.com/user-attachments/assets/40d2bea1-fa19-4ea9-96a5-06a15b62c509">
<img width="1624" alt="image" src="https://github.com/user-attachments/assets/71f33ba9-67d7-4494-bca9-b1ec65f76aa2">

**Comment by ChanYiLin** [comment]
That is expected, the actual size here is the sum of all the snapshots of the volume.
You can get more information from this document: https://longhorn.io/docs/1.7.2/nodes-and-volumes/volumes/volume-size/
Thanks

**Comment by Batyray** [comment]
Thank you for your answer @ChanYiLin  I solved my issue by trimming file system. This is a new concept I just learned is there a document that teaches why file system is filled with unused blocks ?
